### PR TITLE
Remove deprecated update input

### PR DESCRIPTION
**Pull request by angrykoala**
# Description

Remove deprecated top level arguments for nested operations in mutations:

-   create
-   delete
-   connect
-   disconnect
-   connectOrCreate

For example, the following is no longer valid:

_invalid_

```graphql
mutation UpdatePeople {
    updatePeople(create: { movies: { node: { title: "The Good" } } }) {
        people {
            name
        }
    }
}
```

_valid_

```graphql
mutation UpdatePeople {
    updatePeople(update: { movies: { create: { node: { title: "The Good" } } } }) {
        people {
            name
        }
    }
}
```

### CODE REVIEW

File: packages/graphql/tests/e2e/subscriptions/delete-relationship.e2e.test.ts [hunk 1]
```diff
@@ -474,301 +482,6 @@ subscription SubscriptionPerson {
         ]);
     });
 
-    test("disconnect via update - disconnect subscription sends events one way: interface type, by common field", async () => {
```
Comment (darrellwarde) [inline packages/graphql/tests/e2e/subscriptions/delete-relationship.e2e.test.ts:477]: Why the deletion over changing this test here?

**Review by darrellwarde — COMMENTED** [review]
This looks good! I won't comment on every one, but any reason you opted to remove some of these tests instead of updating them?

File: packages/graphql/tests/e2e/subscriptions/delete-relationship.e2e.test.ts (hunk 1 above)
Comment (angrykoala) [reply packages/graphql/tests/e2e/subscriptions/delete-relationship.e2e.test.ts:477]: this was already tested through the other path

**Comment by angrykoala** [comment]
Lots of those are redundant, either because we had duplicate tests, or because now that we have a single path, the other was already tested.

**Pull request merged**